valuePairs.h

```diff
@@ -0,0 +1,9 @@
+typedef struct
+{
```

Comment:
Naming could be better. This struct represents a range and its data. So, something with range would be intuitive.